app-modules/form/src/Filament/Blocks/UrlFormFieldBlock.php

```diff
@@ -56,6 +56,11 @@ public function fields(): array
         return [];
     }
 
+    /**
+     * @param SubmissibleField $field
```

Comment:
Please remove these parameters, the info is already in the type. Only the return types are adding extra value